tests/components/split_to_columns_panel.test.ts

```diff
@@ -0,0 +1,176 @@
+import { Component, onMounted, onWillUnmount, xml } from "@odoo/owl";
+import { Model } from "../../src";
+import { SplitIntoColumnsPanel } from "../../src/components/side_panel/split_to_columns_panel/split_to_columns_panel";
+import { SpreadsheetChildEnv } from "../../src/types";
+import { setCellContent, setSelection } from "../test_helpers/commands_helpers";
+import {
+  click,
+  setCheckboxValueAndTrigger,
+  setInputValueAndTrigger,
+} from "../test_helpers/dom_helper";
+import { mountComponent, nextTick, setGrid, spyModelDispatch } from "../test_helpers/helpers";
+
+interface ParentProps {
+  onCloseSidePanel: () => void;
+}
+class Parent extends Component<ParentProps, SpreadsheetChildEnv> {
+  static components = { SplitIntoColumnsPanel };
+  static template = xml/*xml*/ `
+    <SplitIntoColumnsPanel onCloseSidePanel="props.onCloseSidePanel"/>
+  `;
+  setup() {
+    onMounted(() => this.env.model.on("update", this, () => this.render(true)));
+    onWillUnmount(() => this.env.model.off("update", this));
```

Comment:
> I had this very same problem a while ago. I added a parameter to mountSpreadsheet.

Boolean params are a code smell.
- It makes things much harder to read without knowing by heart what every argument is.
- It's also a sign the function is doing two things (violating the single responsibility principle)
- it increases the cyclomatic complexity because there must be an `if` statement somewhere. Which makes the code again less maintainable/readable

A solution is often to just have two different functions, instead of one with a param. 

:trollface: :trollface: :trollface:

Reply:
hahah I thought exactly the same thing when writing it
At least the boolean is named

Reply:
It doesn't work in my case because calling render on a component  doesn't trigger its own  `onWillUpdateProps` :/